docker/docker-compose.yml

```diff
@@ -36,7 +36,10 @@ services:
           - LOGO=/images/blockscout_logo.svg
           - LOGO_FOOTER=/images/blockscout_logo.svg
           - SUPPORTED_CHAINS=[]
-          - OUTDATED_NETWORK_TITLE=A new zkEVM Testnet is on its way!
-          - OUTDATED_NETWORK_MESSAGE=In the coming days, a new zkEVM Testnet (Pomegranate) will be released to the public, and this version (Litchi) will be outdated. We will update this message with the link to the newest version when it's deployed.
-          - OUTDATED_NETWORK_URL=
+          - SHOW_OUTDATED_NETWORK_MODAL=true
+          - OUTDATED_NETWORK_MODAL_TITLE=Welcome to the new Polygon zkEVM testnet! 
+          - "OUTDATED_NETWORK_MODAL_MESSAGE_1=To connect, follow the instructions available at:"
+          - OUTDATED_NETWORK_MODAL_URL_1=https://public.zkevm-test.net
+          - "OUTDATED_NETWORK_MODAL_MESSAGE_2=The previous network is still available until January 5th. You can access using the information available at:"
```

Comment:
Why `OUTDATED_NETWORK_MODAL_MESSAGE_1` and `OUTDATED_NETWORK_MODAL_MESSAGE_2` are included in the quotes that should belong to the value?

Reply:
Ok... you would not need quotes, but you do because of the colon, and if you add them to the value, the quotes are rendered. Clear.